cache.php

```diff
@@ -4,11 +4,12 @@
 
 class Cache {
 
-	public $cache_folder = '_cache/'; // cache store folder
+	public $cache_folder = '_cache/'; // set cache store folder path
 	public $cache_time = 1 * 60 * 60; // in hour
 
+	public function get_the_json($path, $label, $url) {
+		$this->cache_folder = $path; // override cache store folder path
```

Comment:
There should really be some more data validation here, because the developer implementing this plugin could theoretically define anything as the input `$path` argument, and could potentially break the entire site as a result.